.github/workflows/test-and-lint.yml

```diff
@@ -31,8 +31,8 @@ jobs:
         run: |
           python scripts/download_test_resources.py --debug
       - name: Run tests with Tox
-        # Run tox using the version of Python in `PATH`, and don't include remote resources
-        run: tox -e py -- -k "not REMOTE"
+        # Run tox using the version of Python in `PATH`
+        run: tox -e py
```

Comment:
I think the thing to add here is:
```yaml
- name: Configure AWS Credentials
  uses: aws-actions/configure-aws-credentials@v1
  with:
    aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
    aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
    aws-region: us-west-2
- name: Run tests with Tox
  run: tox -e py
```

Reply:
No need to pass env I don't think as this will dump it to a file. `~/.aws/credentials`

Reply:
Yeah, I was just playing around with this branch to test the workflow, but I totally forgot to include the "Configure AWS Credentials" step. No wonder it wasn't working lol.

Reply:
@JacksonMaxfield I think `aws-actions/configure-aws-credentials` only configures env vars. From the docs [here](https://github.com/marketplace/actions/configure-aws-credentials-action-for-github-actions):
> Configure AWS credential and region environment variables for use in other GitHub Actions. The environment variables will be detected by both the AWS SDKs and the AWS CLI to determine the credentials and region to use for AWS API calls.